pkg/group/repository.go

```diff
@@ -56,5 +55,10 @@ func (r repository) getClusterConfiguration(groupName string) (*model.ClusterCon
 	err := r.db.
 		Where("group_name = ?", groupName).
 		First(&configuration).Error
+	if errors.Is(err, gorm.ErrRecordNotFound) {
```

Comment:
Could we use the `find` we declare above? Then we would not have to repeat the error handling.

Reply:
I don't see how, it's a different entity we're getting